CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 - Added ConVar to toggle double-click buying
 - Added Japanese translation (by @Westoon)
+- Added Drag&Drop role layering VGUI, accessible with the console command `ttt2_edit_rolelayering`
```

Comment:
Please move the changes to the `Unreleased` section again. (Was moved due to a merge/release)

Reply:
Done with https://github.com/TTT-2/TTT2/pull/649/commits/f26eb2e183979dfd5d55f2660101db70d14b766c